plugins/content/shorturl/shorturl.php

```diff
@@ -19,6 +19,10 @@
 require_once JPATH_ROOT . '/components/com_content/helpers/route.php';
 require_once JPATH_ROOT . '/plugins/content/shorturl/helpers/shorturl.php';
 
+if (file_exists(JPATH_ROOT . '/components/com_k2/helpers/route.php')) {
+    require_once JPATH_ROOT . '/components/com_k2/helpers/route.php';
+}
```

Comment:
```
if (file_exists(JPATH_ROOT . '/components/com_k2/helpers/route.php'))
{
	require_once JPATH_ROOT . '/components/com_k2/helpers/route.php';
}
```
Please, use the tab character instead of spaces